client/__tests__/globalSetup.js

```diff
@@ -21,5 +23,4 @@ module.exports = async () => {
   } catch (error) {
     console.error(error);
   }
-  console.log(process.env.TEST_ACCOUNT_PASS);
```

Comment:
Why are you deleting this line?

Reply:
I was accidentally printing the password console <sub><sub>i was really hoping no one would notice</sub></sub>